inspire-atom/src/main/java/org/fao/geonet/services/inspireatom/AtomLocalDescribe.java

```diff
@@ -0,0 +1,95 @@
+package org.fao.geonet.services.inspireatom;
+
+
+import java.nio.file.Path;
+
+import org.apache.commons.lang.StringUtils;
+import org.fao.geonet.Util;
+import org.fao.geonet.constants.Geonet;
+import org.fao.geonet.exceptions.MetadataNotFoundEx;
+import org.fao.geonet.inspireatom.InspireAtomService;
+import org.fao.geonet.inspireatom.util.InspireAtomUtil;
+import org.fao.geonet.kernel.DataManager;
+import org.fao.geonet.kernel.setting.SettingManager;
+import org.fao.geonet.utils.Log;
+import org.jdom.Element;
+
+import jeeves.interfaces.Service;
+import jeeves.server.ServiceConfig;
+import jeeves.server.context.ServiceContext;
+
+/**
+ * Service to get local Atom feed.
+ *
+ * @author Gustaaf Vandeboel
+ */
+public class AtomLocalDescribe implements Service {
```

Comment:
We are now trying to use Spring MVC only for new services. So migrating this to spring would be nice to have.